platform/darwin/src/MGLStyle.h

```diff
@@ -263,6 +263,13 @@ MGL_EXPORT
  */
 @property (nonatomic, assign) BOOL performsPlacementTransitions;
 
+/**
+A set containing user-specified source identifiers for point features available for accessibility. The point features must have a `name` attribute that matches those specified by <a href="https://www.mapbox.com/vector-tiles/mapbox-streets-v8/#overview">Mapbox Streets</a> source and belong to a `MGLVectorStyleLayer`.
+ 
+This set does not include Mapbox Streets source identifiers included by default.
+*/
+@property (nonatomic) NSSet *accessiblePlaceSourceIdentifiers;
```

Comment:
The type should be qualified with the type of each element: `NSSet <NSString *> *`.